# close in after

You can use the `after` block to exit gracefully whether the code in `try` was successful or not. Please use the `after` block to close `resource`.
